apps/web-giddh/src/app/new-inventory/component/stock-create-edit/stock-create-edit.component.ts

```
@@ -1218,6 +1218,8 @@ export class StockCreateEditComponent implements OnInit, OnDestroy {
             delete variant.fixedAssetAccountDetails;
             delete variant.purchaseAccountDetails;
             delete variant.salesAccountDetails;
+            delete variant.attachmentName;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)